docs/index.rst

```diff
@@ -45,6 +45,7 @@ Publicly available data access and web services are currently in development.
    query/Getting Started.ipynb
    query/List Products.ipynb
    query/Loading Data.ipynb
+   internal/new_product.rst
```

Comment:
Start a new section (eg Developer Guide) to keep this level of detail away from the User Guide.